src/components/EventItem.js

```diff
@@ -38,7 +38,7 @@ const EventItem = ( props ) => {
 					</div>
 					{ venue && (
 						<div>
-							Venue: { venue }
+							<span class="dashicons dashicons-location">&nbsp;</span> { venue }
```

Comment:
Take a look at styling done for the dashicons on the editor so you don't need to use `&nbsp;`. I think they are flex layouts if I recall.

Reply:
Also, let's put this after the title rather than before. Thx!